Makefile

```diff
@@ -20,21 +28,40 @@ all: fixtures
 	$(warning The `all` target is deprecated. Use `fixtures` instead.)
 
 fixtures: fixtures/docker \
+	fixtures/drpm-unsigned \
```

Comment:
Nitpick: all of the other lines below use spaces, and using a tab character here is out-of-place.